content/docs/command-reference/list.md

```diff
@@ -68,10 +70,12 @@ files recursively.
 - `-v`, `--verbose` - displays detailed tracing information. when this option is
   not specified.
 
-## Example: List files and directories in a DVC repository
+## Example: Find files to download from a repository
```

Comment:
may be we want import them?

Reply:
Yes, dvc import is mentioned in the paragraph below. I don't understand the implication or suggestion in terms of this header change.

Reply:
`to download` a bit to narrow in this case ... I can list them for different purposes

Reply:
This example uses list and then get, that's why the header says "download". It doesn't mean to describe what list is for, that's done earlier in the description.

Reply:
No a big deal, @jorgeorpinel 👍 .. up to you :)